drop-boxes/register-raw-and-mzML/register-raw-and-mzml.py

```diff
@@ -216,8 +216,13 @@ def process(transaction):
     mzml_path = None
     openbis_format_code = None
     for root, subFolders, files in os.walk(incomingPath):
-        if subFolders:
-            subFolder = subFolders[0]
+        if s in subFolders:
+            stem, ext = os.path.splitext(s)
+            if ext.lower() in VENDOR_FORMAT_EXTENSIONS:
+                if raw_path:
+                    raise ValueError("More than one raw file found. Only one pair of raw data and mzML can be registered at a time")
```

Comment:
Question: I believe this exception will never happen since raw_path is set to None in line 215 and not updated before it's checked here 🤔

Reply:
good point, it was supposed to be a for loop, not an "if"